easybuild/easyconfigs/i/ifort/ifort-2015.5.223-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,23 @@
+name = 'ifort'
+version = '2015.5.223'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "Fortran compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_fcompxe_%(version)s.tgz']
+
+gcc = 'GCC'
+gccver = '4.9.3-2.25'
+versionsuffix = '-%s-%s' % (gcc, gccver)
```

Comment:
it's probably cleaner to stick to using `GCC` as a label in the versionsuffix, but using `GCCcore` as a dep? @ocaisa?